### ISSUE TITLE
Make GitHub notify when an eval error has occoured?

### ISSUE THREAD

**Issue by doronbehar** [NixOS/ofborg#652]
Rather often, when you rename a package attribute or alike, another package you were not aware of has used the attribute name you deleted, and you forgot to change it accordingly.

It'd be nice if ofborg had sent via GitHub an email notifying of the eval failure - just like the EditorConfig job for example. This is helpful in slow evaluation which happen in e.g `treewide:` like PRs.

**Comment by Artturin** [comment]
How can ofborg know that the attr a pr touches isn't available anymore unless the pr is re-evaluated

**Comment by pbsds** [comment]
You do get a red X in the https://github.com/pulls view when CI fails. Sending messages every time eval fails would cause a storm everytime master fails eval.

**Comment by doronbehar** [comment]
> You do get a red X in the [github.com/pulls](https://github.com/pulls?q=is%3Apr+is%3Aopen+author%3A%40me+archived%3Afalse+sort%3Aupdated-desc) view when CI fails. Sending messages every time eval fails would cause a storm everytime master fails eval.

Correct, but I don't think it is bad. Eval errors on `master` happen rarely, and when they do, I think it wouldn't hurt to notify many people about it, as it is a severe mistake. Also, even if the error is not due to merges you personally performed, it will also give you the hint to comment `@ofborg eval` to retrigger evaluation after the eval error is fixed.

**Comment by pbsds** [comment]
Doesn't this work?
![image](https://github.com/NixOS/ofborg/assets/140964/31f05bb3-240f-48d6-b376-1ca6a1107674)

**Comment by doronbehar** [comment]
@pbsds this seems worth checking, but I don't understand how did you get to that interface? I also wonder, why is EditorConfig's workflow notifying me upon emails and ofborg doesn't? The interface you screenshot doesn't look like it has different settings for the above 2 workflows...

**Comment by pbsds** [comment]
It may be because the editor config checks run in a github action runner, while the ofborg CI checkmarks are set using some other API by the ofborg runners.

**Comment by wolfgangwalther** [comment]
Since eval checks are now happening in nixpkgs' GHA workflows, we also get notified about them.
